src/app/views/marketplace/marketplace-app.controller.coffee

```diff
@@ -41,6 +41,14 @@ angular.module 'mnoEnterpriseAngular'
 
         vm.isLoading = false
 
+      # Update the comments and avergae review
+      vm.updateReview = (newReview) ->
+        # Select and push to comments the last comment added
+        [first, ..., last] = newReview.app.reviews
```

Comment:
Also replace
`[first, ..., last] = newReview.app.reviews`
by
`[..., newReview] = response.app.reviews` 

And use `newReview` instead of `last`